### PR TITLE
diphotons bkgs for 2017

### PR DESCRIPTION
**Pull request by gourangakole**

### CODE REVIEW

**Comment by perrozzi** [comment]
please apply these suggestions
https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

**Comment by gourangakole** [comment]
Sorry, now this should be ok. 
Could you please check once.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/diphotonJets_5f_NLO_FXFX_Mgg80ToInf/diphotonJets_5f_NLO_FXFX_Mgg80ToInf_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import model loop_sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model loop_sm-no_b_mass
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/diphotonJets_5f_NLO_FXFX_Mgg80ToInf/diphotonJets_5f_NLO_FXFX_Mgg80ToInf_proc_card.dat]: this needs to go

Comment (gourangakole) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/diphotonJets_5f_NLO_FXFX_Mgg80ToInf/diphotonJets_5f_NLO_FXFX_Mgg80ToInf_proc_card.dat]: Just to be clear:
I kept "import model loop_sm-ckm_no_b_mass" line 
and removed other commented lines.

**Comment by perrozzi** [comment]
@pmillet can you please check the sherpa datacard?

**Comment by pmillet** [comment]
@perrozzi  @gourangakole  
looks good to me
if i remember correctly we directly got the card from the authors

**Pull request merged**